src/libsync/account.h

```diff
@@ -75,6 +75,9 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject {
     void setSharedThis(AccountPtr sharedThis);
     AccountPtr sharedFromThis();
 
+    /// The user that can be used in dav url
```

Comment:
I  don't know exactly.  the answer from @DeepDiver1975 on #engineering (17:26) was not enough for me to make a better comment.